manim/mobject/mobject.py

```diff
@@ -2600,7 +2600,14 @@ def construct(self):
     def interpolate_color(self, mobject1, mobject2, alpha):
         raise NotImplementedError("Please override in a child class.")
 
-    def become(self, mobject: "Mobject", copy_submobjects: bool = True):
+    def become(
+        self,
+        mobject: "Mobject",
+        copy_submobjects: bool = True,
+        match_center: bool = False,
+        match_height: bool = False,
+        match_width: bool = False,
+    ):
         """Edit points, colors and submobjects to be identical
         to another :class:`~.Mobject`
 
```

Comment:
Can you please add a Parameters section to the docstring where the new keyword arguments are documented?